### PR TITLE
Add bdd tests for update

### PR DESCRIPTION
**Pull request by TengZhang19**
add bdd tests for update in customer.feature

### CODE REVIEW

File: features/customer.feature [hunk 1]
```diff
@@ -90,4 +90,28 @@ Scenario: Delete a Customer
     Then I should see the message "Customer has been Deleted!"
     When I press the "Clear" button
     And I press the "Search" button
-    Then I should not see "fn1" in the results
\ No newline at end of file
+    Then I should not see "fn1" in the results
+
+Scenario: Update a Customer
+    When I visit the Home Page
+    And I set the "first_name" to "fn2"
```
Comment (shcai97) [inline features/customer.feature:97]:
```suggestion
    And I set the "First name" to "fn2"
```

Comment (shcai97) [reply features/customer.feature:97]: Can use naming convention after merging #96 
It will improve readbility

File: features/customer.feature [hunk 2]
```diff
@@ -90,4 +90,28 @@ Scenario: Delete a Customer
...
+    And I should see "fn2" in the results
+    When I set the "user_id" to "222"
+    And I set the "first_name" to "fn2_updated"
+    And I set the "last_name" to "ln2_updated"
+    And I set the "password" to "123456"
+    And I set the "apartment" to "a2_updated"
+    And I set the "street" to "s2_updated"
+    And I set the "city" to "c2_updated"
+    And I set the "state" to "se2_updated"
+    And I set the "zip_code" to "zip2_updated"
```
Comment (shcai97) [inline features/customer.feature:109]:
```suggestion
    And I set the "ZIP code" to "zip2_updated"
```

File: features/customer.feature [hunk 3]
```diff
@@ -90,4 +90,28 @@ Scenario: Delete a Customer
...
\ No newline at end of file
+    Then I should not see "fn1" in the results
+
+Scenario: Update a Customer
+    When I visit the Home Page
+    And I set the "first_name" to "fn2"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    And I should see "fn2" in the results
+    When I set the "user_id" to "222"
```
Comment (shcai97) [inline features/customer.feature:101]:
```suggestion
    When I set the "User ID" to "222"
```

**Review by shcai97 — CHANGES_REQUESTED** [review]
Can use naming convention to improve readability.

File: features/customer.feature (hunk 3 above)
Comment (TengZhang19) [reply features/customer.feature]: nice catch, would fix them asap

**Review by shcai97 — APPROVED** [review]

**Pull request merged**